test/org/zaproxy/zap/extension/pscanrulesAlpha/BlankLinkTargetScannerUnitTest.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2018 The ZAP development team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.pscanrulesAlpha;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+import org.parosproxy.paros.network.HttpMalformedHeaderException;
+import org.parosproxy.paros.network.HttpMessage;
+import org.zaproxy.zap.extension.pscan.PluginPassiveScanner;
+
+public class BlankLinkTargetScannerUnitTest extends PassiveScannerTest {
+
+    @Override
+    protected PluginPassiveScanner createScanner() {
+        return new BlankLinkTargetScanner();
+    }
+    
+    @Test
+    public void noLinks() throws HttpMalformedHeaderException {
+
+        HttpMessage msg = new HttpMessage();
+        msg.setRequestHeader("GET https://www.example.com/test/ HTTP/1.1");
+
+        msg.setResponseBody("<html></html>");
+        msg.setResponseHeader(
+                "HTTP/1.1 200 OK\r\n" + 
+                "Content-Type: text/html;charset=ISO-8859-1\r\n" + 
+                "Content-Length: " + msg.getResponseBody().length() + "\r\n");
```

Comment:
It seems that setting the request/response headers could be extracted to a utility method instead of repeating the same code in every UnitTest.